resources/util_test.go

```diff
@@ -378,3 +378,45 @@ func TestCurrentUserGroupById(t *testing.T) {
 		t.Errorf("gid didn't match current user's: %s vs %s", strconv.Itoa(gid), currentGID)
 	}
 }
+
+func TestStrSortedSliceCompare0(t *testing.T) {
+	slice1 := []string{"foo", "bar", "baz"}
```

Comment:
@purpleidea Since you haven't merged this yet, I'm going to fix these indices so they all match (slice0, slice1.) Won't make any other changes. :)